src/java/test/org/apache/zookeeper/server/NettyServerCnxnTest.java

```diff
@@ -51,9 +65,9 @@ public void setUp() throws Exception {
      * 
      * @see <a href="https://issues.jboss.org/browse/NETTY-412">NETTY-412</a>
      */
-    @Test(timeout = 40000)
+    @Test(timeout = 30000)
```

Comment:
Due to @fpj previous review on previous incarnation this patch: https://github.com/apache/zookeeper/pull/77/files#r80003749 (sorry, should have put the original link).